lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -581,6 +583,7 @@ sub job_grab {
 
     # TODO: cleanup previous tmpdir
     $worker->set_property('WORKER_TMPDIR', tempdir());
+    OpenQA::IPC->ipc->emit_signal('scheduler', 'job_grab', $job_hashref);
```

Comment:
this would also emit secret properties such as jobtoken, wouldn't it? Maybe it should just emit the job id instead